docs/intro/tutorial01.txt

```diff
@@ -216,7 +392,7 @@ database and type ``\dt`` (PostgreSQL), ``SHOW TABLES;`` (MySQL), or
 The development server
```

Comment:
The introduction of the development server could go after creating the project and before creating the app. Then we can keep the "Welcome to Django" bit.